worker-document-archetype/src/main/resources/archetype-resources/__rootArtifactId__-container/pom.xml

```diff
@@ -84,6 +83,55 @@
 
     <build>
         <plugins>
+            <!-- Unpack the base worker configuration files -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>process-config-files</id>
+                        <goals>
+                            <goal>unpack</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>com.github.workerframework</groupId>
+                                    <artifactId>worker-default-configs</artifactId>
+                                    <outputDirectory>${project.build.directory}/config-files</outputDirectory>
+                                    <excludes>**/META-INF/**</excludes>
```

Comment:
Change the excludes line to:

    <excludes>META-INF/**,cfg*StorageServiceDataStore*</excludes>